docs/book/src/migration/v2vsv3.md

```diff
@@ -2,21 +2,71 @@
 
 This document covers all breaking changes when migrating from v2 to v3.
 
-The details of all changes (breaking or otherwise) can be found in [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/releases) release notes.
+The details of all changes (breaking or otherwise) can be found in
+[controller-runtime](https://github.com/kubernetes-sigs/controller-runtime/releases),
+[controller-tools](https://github.com/kubernetes-sigs/controller-tools/releases)
+and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/releases)
+release notes.
+
+## Common changes
+V3 project uses go modules and the go version 1.15 wheredep is no longer supported.
 
 ## Kubebuilder
 
 - A plugin design was introduced to the project. For more info see the [Extensible CLI and Scaffolding Plugins][plugins-phase1-design-doc]
+
 - The GO supported version was upgraded from 1.13+ to 1.15+
 
-## Project config versions
+- New PROJECT file layout which stores the data used to do the scaffolds with the purpose to support the new features and allow future the CLI provide helpers to it users in the future. 
+
+## New plugin go/v3
```

Comment:
we should give a brief rundown of plugins vs project version above so this section makes sense.

Also, maybe change the title around a bit, like

```suggestion
## Notable changes in the go/v3 plugin
```

or the following (more in line with the slightly casual voice we take in the KB docs):

```suggestion
## TL;DR of the New `go/v3` Plugin

***More details on this can be found at [insert link here], but for the highlights, check below***
```